src/mscorlib/src/System/String.Comparison.cs

```diff
@@ -792,6 +792,25 @@ public int CompareTo(String strB)
             return false;
         }
 
+        [Pure]
+        public Boolean EndsWith(Char value)
+        {
+            return EndsWith(new string(value, 1), StringComparison.CurrentCulture);
```

Comment:
> Do we actually need a culture sensitive EndsWith char overload?

I asked the same thing in the issue: https://github.com/dotnet/corefx/issues/4805#issuecomment-270718228